modules/preparing-for-installing-microshift-rpm.adoc

```diff
@@ -1,37 +1,37 @@
 // Module included in the following assemblies:
 //
-// microshift/microshift-install-rpm.adoc 
+// microshift/microshift-install-rpm.adoc
 
 [id="preparing-install-microshift-from-rpm-package_{context}"]
 = Preparing to install {product-title} from an RPM package
 
 Before installing {product-title} from an RPM package, you must configure your {op-system} machine to have a logical volume manager (LVM) volume group (VG) with sufficient capacity for the persistent volumes (PVs) of your workload.
 
-{product-title} uses the logical volume manager storage (LVMS) Container Storage Interface (CSI) provider for storing PVs. LVMS relies on Linux's LVM to dynamically manage the backing storage for PVs. For this reason, your machine must have an LVM VG in which LVMS can create the LVM logical volumes (LVs) for your workload's PVs.  
+{product-title} uses the logical volume manager storage (LVMS) Container Storage Interface (CSI) provider for storing PVs. LVMS relies on Linux's LVM to dynamically manage the backing storage for PVs. For this reason, your machine must have an LVM VG with unused space in which LVMS can create the LVM logical volumes (LVs) for your workload's PVs.
 
-To configure an LVM VG that allows LVMS to create the LVM LVs for your workload's PVs, adjust your root volume's size during the installation of {op-system}. Adjusting your root volume's size provides free space for additional LVs created by LVMS at runtime. 
+To configure an LVM VG that allows LVMS to create the LVM LVs for your workload's PVs, adjust your root volume's size during the installation of {op-system}. Adjusting your root volume's size provides free space for additional LVs created by LVMS at runtime.
```

Comment:
```suggestion
To configure a volume group that allows LVMS to create the logical volumes for your workload's PVs, adjust your root volume's size during the installation of {op-system}. Adjusting your root volume's size provides free space for additional LVs created by LVMS at runtime.
```

nit: There's  nothing technically wrong with the current text, but it's not necessary to specify that a volume group or physical volume is related to LVM (they are by definition). Spelling the words out feels clearer.

Reply:
Agreed, spelled out volume group and defined (VG), but we defined LVs just above there, so I made that one just LVs.